packages/jaeger-ui/src/components/TracePage/TraceTagOverview/PopupSQL.css

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
For consistency (e.g. with `UiFindInput.*`), the file should be named `PopupSql.css`